.gitignore

```diff
@@ -1,5 +1,7 @@
 data/*
 example_data/*
```

Comment:
what you're using as `/test_data` I use as `/example_data`. We shouldn't post example files up on github, since they're quite space-consuming.

Reply:
I used a shortened version of the files, so it's 5.4 mb total, but I can still add it to .gitignore if that's too large.

Reply:
That's not too large -- can I ask where those test files come from?

Reply:
There are 3 input files, the rest in the folder were generated by the script. 

One is a shortened version of the leaned motifmap file you made. Another is the entire reference file. Last is an example peakset from NeuroLINCS data. We can generate a random BED file if we don't want to publish. The first two were taken from `/nfs/latdata/alex/garnet_data/old/`

Reply:
gotcha. I think using a fraction of a peakset from a neurolincs patient sounds safe to me. If it isn't difficult though, I think it would be even more kosher to use some random dataset out there... I'll leave that up to you.

Reply:
Need to go offline rn, will finish these changes this weekend.

Reply:
Changed to only chromosome 1 peak file